components/product-details/index.js

```diff
@@ -3,46 +3,57 @@ import personData from '../../mock/person.json';
 import PropTypes from 'prop-types';
 import path from 'path';
 import Image from 'next/image';
+import Link from 'next/link';
+import { Footer } from '../../components/footer';
+import styles from './product-details.module.css';
 
 export const ProductDetails = ({ productJSON }) => {
   return (
     <>
       <NavBar personData={personData} />
-      <div className="product-container">
-        <div className="product-image">
+      <div className={styles.details}>
+        <div className={styles.bigImg}>
           <Image
+            width={600}
+            height={319}
             src={path.join('/assets', productJSON.image)}
             alt={productJSON.id}
-            width={200}
-            height={200}
+            layout="responsive"
           />
         </div>
 
-        <div className="product-details">
-          <h2> {productJSON.name}</h2>
-          <h3> Where to use</h3>
-          <ul>
-            {productJSON.usage.map((use, index) => {
-              return <li key={index}>{use}</li>;
-            })}
-          </ul>
+        <div className={styles.box}>
+          <div className={styles.row}>
+            <h2>{productJSON.name}</h2>
+            <span>{productJSON.price} Gold Coins</span>
+
+            <h3 className={styles.description}>Description</h3>
+            <p>
+              {productJSON.usage.map((use, id) => {
+                return <p key={id}> {use}</p>;
+              })}
```

Comment:
NIT: we can create a new function `renderUses` and pass this `productJSON.usgae` as an argument and that function will return this.
Good practise and that will make this code more readable.

Reply:
Yes, Done.